test.disabled/ejabberd_tests/tests/connect_SUITE.erl

```diff
@@ -37,34 +37,40 @@
 %%--------------------------------------------------------------------
 
 all() ->
-    [{group, c2s_noproc}, %% should be first, uses vanilla config
-     {group, starttls},
-     {group, feature_order},
-     {group, tls}
+    [ {group, fast_tls}
+%     ,{group, just_tls}
     ].
 
 groups() ->
-    [{c2s_noproc, [], [reset_stream_noproc,
-                       starttls_noproc,
-                       compress_noproc,
-                       bad_xml,
-                       invalid_host,
-                       invalid_stream_namespace,
-                       pre_xmpp_1_0_stream]},
-     {starttls, test_cases()},
-     {tls, [parallel], [auth_bind_pipelined_session,
-                        auth_bind_pipelined_auth_failure |
-                        generate_tls_vsn_tests() ++ cipher_test_cases()]},
-     {feature_order, [parallel], [stream_features_test,
-                                  tls_authenticate,
-                                  tls_compression_fail,
-                                  tls_compression_authenticate_fail,
-                                  tls_authenticate_compression,
-                                  auth_compression_bind_session,
-                                  auth_bind_compression_session,
-                                  bind_server_generated_resource]}
+    [ {c2s_noproc, [], [reset_stream_noproc,
```

Comment:
we don't need to touch the whole block. 
unless you want to create functions feature_order_cases(), c2s_noproc_cases(), .....